CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 12.1.3 "ruby-racecar" (2021-07-21)
+
+### material
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **input:** show focus indication for readonly inputs ([#22847](https://github.com/angular/components/issues/22847)) ([afa943101f3757a66b76e8475da9468b49295123](https://github.com/angular/components/commit/afa943101f3757a66b76e8475da9468b49295123)), closes [#22783](https://github.com/angular/components/issues/22783) |
+| bug fix |  **multiple:** incosistent item text truncation when used together with MDC list ([#23138](https://github.com/angular/components/issues/23138)) ([48739eb0962188a1adbb3ac9bef6b00d44f6c2a8](https://github.com/angular/components/commit/48739eb0962188a1adbb3ac9bef6b00d44f6c2a8)) |
```

Comment:
This one should be under material-experimental also also there's a typo in the commit message ("incosistent")